src/utils/urlTransform.ts

```diff
@@ -0,0 +1,25 @@
+function transformGitHubApiUrl(apiUrl?: string) {
+    if (!apiUrl) return '#';
+    const transformedUrl = apiUrl.replace(
+        /https:\/\/api\.github\.com\/repos\//,
+        'https://github.com/'
+    );
+
+    return transformedUrl;
+}
+
+function extractRepoName(issueUrl: string) {
+    const match = issueUrl.match(
+        /(?:https:\/\/github\.com\/([^/]+)\/([^/]+)\/issues\/(\d+))|(?:https:\/\/api\.github\.com\/repos\/([^/]+)\/([^/]+)\/issues\/(\d+))/
+    );
+
+    if (match) {
+        const repo = match[2] || match[5];
+        const issueNumber = match[3] || match[6];
+        return `${repo.replace(/-/g, ' ')} #${issueNumber}`;
+    }
+
+    return null;
+}
```

Comment:
Please add 1-2 examples that how we're using this

Reply:
Supplying comments will be more helpful to understand what is being done here

Reply:
Okay, I see the test case and that clears things.